content/pom.xml

```diff
@@ -272,7 +272,7 @@
                             <execution>
                                 <phase>prepare-package</phase>
                                 <configuration>
-                                    <tasks>
+                                    <target>
```

Comment:
Is this some accidental modification?

Reply:
When trying to install core-components using the cloud profile for sdk instance, I got the error that the maven-antrun-plugin changed and now is using target instead of tasks. This error should appear for everyone when installing the core-components with this command: mvn clean install -PautoInstallSinglePackage,cloud
https://maven.apache.org/plugins/maven-antrun-plugin/

Reply:
Fixed/reverted with https://github.com/adobe/aem-core-wcm-components/pull/2723/commits/b152ea1e435b32b97f75d3d3279cfe6e683a5b76

Explicitly specifying the plugin version avoids the issue in the newer/latest versions